vite.sdk-assets-config.js

```diff
@@ -36,7 +36,7 @@ export default mergeOptions(commonOptions, {
     plugins: [
         themeBuilder({
             themeConfig: {
-                themes: { element: "./src/platform/web/ui/css/themes/element" },
+                themes: ["./src/platform/web/ui/css/themes/element"],
```

Comment:
I wonder if this should go in `vite.common-config.js` since it's common. Will have to think on Monday.

Reply:
I guess the problem then would be that it also would add the plugin for the sdk library build?

Reply:
Ah yes!